school/rental.rb

```diff
@@ -0,0 +1,14 @@
+class Classroom
```

Comment:
- [ ] Great job. kindly add a rental class
- [ ] kindly add its setters and getters for the @date instance variable, and ensure that the @date instance variable is initialized in the constructor